Projects/Foundations/Schemas/Docs-Nodes/D/Data/Data-Tasks/data-tasks.json

```diff
@@ -65,35 +105,27 @@
                     "language": "RU",
                     "text": "Вы также можете настроить задачи интеллектуального анализа данных, используя различные функции в этом разделе иерархии, начиная с параметров в меню этого узла. Этот маршрут предлагает более точный контроль над тем, какие задачи вы настраиваете и как.",
                     "updated": 1638355323726
+                },
+                {
+                    "language": "TR",
+                    "text": "Bu düğümün menüsündeki seçeneklerden başlayarak hiyerarşinin bu bölümündeki farklı işlevleri kullanarak veri madenciliği görevlerini de ayarlayabilirsiniz. Bu yol, hangi görevleri nasıl ayarlayacağınız konusunda daha hassas bir kontrol sunar.",
+                    "updated": 1654391840348,
+                    "style": "Text"
                 }
             ]
         },
         {
             "style": "Include",
             "text": "Foundations->Concept->Reusable Snippets->Deleting Node Warning",
-            "updated": 1610122758940
+            "updated": 1610122758940,
+            "translations": [
+                {
+                    "language": "TR",
+                    "text": "Foundations->Concept->Reusable Snippets->Düğüm Silme Uyarısı",
+                    "updated": 1654391847059,
+                    "style": "Include"
+                }
+            ]
         }
-    ],
-    "definition": {
-        "text": "Data tasks referes to data-mining, that is, the activity of processing data. You need to process data to feed charts, and so that your trading systems may make decisions based on quality information.",
-        "updated": 1609618235002,
-        "translations": [
-            {
-                "language": "ES",
-                "text": "Data tasks refiere a la minería de datos, es decir, la actividad de procesamiento de datos. Necesitarás procesar datos para alimentar gráficos y para que sus sistemas de trading puedan tomar decisiones basadas en información de calidad.",
-                "updated": 1624283536808,
-                "style": "Definition"
-            },
-            {
-                "language": "RU",
-                "text": "Data Tasks относятся к интеллектуальному анализу данных, т.е. к деятельности по обработке данных. Вам необходимо обрабатывать данные для подачи графиков, чтобы Ваши торговые системы могли принимать решения на основе качественной информации.",
-                "updated": 1638356535434
-            },
-            {
-                "language": "TR",
-                "text": "Veri görevleri, veri madenciliğini, yani veri işleme faaliyetini ifade eder. Grafikleri beslemek için verileri işlemeniz gerekir ve böylece ticaret sistemleriniz kaliteli bilgilere dayalı kararlar verebilir.",
-                "updated": 1639120026724
-            }
-        ]
-    }
```

Comment:
Is this related to the translations added above?

Reply:
The last 2 comments you posted are just moving translations from the wrong place to the right place. Not an addition or deletion.